src/Test/AtomicTests/constraints.jl

```diff
@@ -0,0 +1,24 @@
+"""
+    Test getting constraints by name.
+"""
+function getconstraint(model::MOI.ModelLike, config::TestConfig)
+    MOI.empty!(model)
```

Comment:
It's weird that we have to call `MOI.empty!` at the beginning of every test. This should happen somewhere else.

Reply:
I agree. Maybe it should go here?
https://github.com/JuliaOpt/MathOptInterface.jl/blob/84c5b19fa7cea292b715f6e02658bf977db08134/src/Test/config.jl#L42-L43

That can be a job for a different PR.